src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/PublicationServiceTests.cs

```diff
@@ -288,6 +288,168 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
+                var result = await service.Get(publication.Slug);
+
+                var publicationViewModel = result.AssertRight();
+                
+                Assert.Equal(publication.Id, publicationViewModel.Id);
+                Assert.True(publicationViewModel.IsSuperseded);
+                
+                Assert.Equal(publication.SupersededBy.Title, publicationViewModel.SupersededBy?.Title);
+                Assert.Equal(publication.SupersededBy.Slug, publicationViewModel.SupersededBy?.Slug);
+                Assert.Equal(publication.SupersededBy.Id, publicationViewModel.SupersededBy!.Id);
+            }
+        }
+
+        [Fact]
+        public async Task Get_PublicationTree_SupersededPublicationBuildsPublicationSupersededByViewModel()
+        {
+            var releaseA = new Release
+            {
+                ReleaseName = "2020",
+                TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                Published = DateTime.UtcNow
+            };
+            var releaseB = new Release
+            {
+                ReleaseName = "2020",
+                TimePeriodCoverage = TimeIdentifier.CalendarYear,
+                Published = DateTime.UtcNow
+            };
+            
+            var publicationA = new Publication
+            {
+                Title = "Publication A",
+                Slug = "publication-a",
+                LatestPublishedRelease = releaseA,
+                LatestPublishedReleaseId = releaseA.Id,
+                SupersededBy = new Publication
+                {
+                    Id = Guid.NewGuid(),
+                    Title = "supersede publication",
+                    Slug = "supersede-publication",
+                    LatestPublishedReleaseId = Guid.NewGuid()
+                },             
+                Releases = new List<Release>
+                {
+                    releaseA
+                },
+            };
+            
+            var publicationB = new Publication
+            {
+                Title = "Publication B",
+                Slug = "publication-b",
+                LatestPublishedRelease = releaseB,
+                Releases = new List<Release>
+                {
+                    releaseB
+                }
+            };
+
+            var themes = new List<Theme>
+            {
+                new()
+                {
+                    Title = "Theme A",
+                    Summary = "Theme A summary",
+                    Topics = new List<Topic>
+                    {
+                        new()
+                        {
+                            Title = "Topic A",
+                            Publications = ListOf(publicationA)
+                        },
+                        new()
+                        {
+                            Title = "Topic B",
+                            Publications = ListOf(publicationB)
+                        }
+                    }
+                }
+            };
```

Comment:
This test only needs to check that the superseded view model is set so there's no need to set up two different publications. One of the publications isn't even published so isn't in the result for you to check that both have the superseded view model.

I would remove one of the publication / release / and topic. It looks like it's `topicA`, `publicationA`, `releaseA` which is not published and can be removed, allowing you then to rename all the 'B's' when you've only got one of each.

Reply:
I've added `PublicationServiceTests#Get_IsSuperseded_SupersedingPublicationBuildsSupersededViewModel` to replace this